docs/docs/install-shells.mdx

```diff
@@ -23,7 +23,11 @@ oh-my-posh --print-shell
 }>
 <TabItem value="powershell">
 
-Edit `$PROFILE` in your preferred PowerShell version and add the following line.
+Edit your PowerShell profile script, you can find its location under the `$PROFILE` variable in your preferred PowerShell version. For example, using notepad:
+```powershell
```

Comment:
didn't see this but the linter will fail due to lack of a space between the paragraph and code block.

Reply:
fixed